src/store/modules/toasts.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  namespaced: true,
+  actions: {
+    show ({ commit }, toast) {
+      // The corresponding plugin takes care of the rest.
```

Comment:
Why use a plugin when you can directly write the code in here?

Reply:
Because I didn't want to import quasar directly into a store module.
I thought it might be a bit cleaner if I do the quasar stuff in a plugin...

Reply:
There's no big difference, when someone does `import store from "@/store" this also loads all plugins. 

Maybe it would be slightly simpler to test because one could load the module without the plugin, on the other hand it's quite easy to mock the Quasar dependency.

Reply:
So should I refactor it to remove the plugin? For me, both options are okay.

Reply:
No strong need to rewrite, and the comment explains it well why the action is empty. With the rewrite, we wouldn't need a comment at all though ;)